api/v1alpha1/servicebinding_types.go

```diff
@@ -91,7 +92,7 @@ type ServiceBindingSpec struct {
 	// BindAsFiles makes available the binding values as files in the application's container
 	// See MountPath attribute description for more details.
 	// +optional
-	BindAsFiles bool `json:"bindAsFiles,omitempty"`
+	BindAsFiles *bool `json:"bindAsFiles,omitempty"`
```

Comment:
CRD needs to be regenerated

Reply:
I see this error when I try to regenerate CRDs:
```
➜  service-binding-operator git:(default-projection) ✗ operator-sdk generate crds   
INFO[0000] Running CRD generator.                       
-: pattern ./pkg/apis/...: lstat ./pkg/apis/: no such file or directory
FATA[0000] error generating CRDs from APIs in pkg/apis: error generating CRD manifests: error running CRD generator: not all generators ran successfully
```

Reply:
with migrating to operator-sdk 1.3, we do this by invoking `make generate`